Changelog.md

```diff
@@ -0,0 +1,102 @@
+# Changelog
+
+## 1.0.0 (2022-04-06)
```

Comment:
This code has been running stable for several years. Time to acknowledge it as stable :)